pkg/application/consul/consul.go

```diff
@@ -0,0 +1,82 @@
+package consul
+
+import (
+	"github.com/awslabs/eksdemo/pkg/application"
+	"github.com/awslabs/eksdemo/pkg/cmd"
+	"github.com/awslabs/eksdemo/pkg/installer"
+	"github.com/awslabs/eksdemo/pkg/template"
+)
+
+// https://developer.hashicorp.com/consul/docs/k8s/installation/install
+// https://developer.hashicorp.com/consul/tutorials/kubernetes/kubernetes-eks-aws
+// https://artifacthub.io/packages/helm/hashicorp/consul
+
+func NewApp() *application.Application {
+	app := &application.Application{
+		Command: cmd.Command{
+			Name:        "consul",
+			Description: "HashiCorp Consul Service-Mesh",
+		},
+
+		Installer: &installer.HelmInstaller{
+			ChartName:     "consul",
+			ReleaseName:   "consul",
+			RepositoryURL: "https://helm.releases.hashicorp.com",
+			VersionField:  "1.4.1",
+			ValuesTemplate: &template.TextTemplate{
+				Template: valuesTemplate,
+			},
+		},
+	}
+
+	app.Options, app.Flags = newOptions()
+
+	return app
+
+}
+
+const valuesTemplate = `---
+global:
+  # The main enabled/disabled setting.
+  # If true, servers, clients, Consul DNS and the Consul UI will be enabled.
+  # Namespace
+  namespace: {{ .Namespace }}
```

Comment:
I think these lines can be deleted. I didn't see a `global.namespace` setting in https://github.com/hashicorp/consul-k8s/blob/main/charts/consul/values.yaml. and the first 2 comments are for a setting `global.enabled` which isn't included.

Reply:
put enabled = true as a default back in. I may need to revisit this, and adjust the bool if --enable-ui is used. It looks like the consul-ui is enabled as a ClusterIP now by default.